spec/index.bs

```diff
@@ -569,6 +561,63 @@ enum GPUBindingType {
 };
 </script>
 
+  * {{GPUBindGroupLayoutBinding/textureDimension}}, {{GPUBindGroupLayoutBinding/multisampled}}:
+    The dimensions and multi-sampling properties of texture bindings.
+
+    Note: This allows Metal-based implementations to back the respective bind
+    groups with `MTLArgumentBuffer` objects that are more efficient to bind at
+    run-time.
+
+  * {{GPUBindGroupLayoutBinding/dynamic}}:
+    For [=uniform-buffer=], [=storage-buffer=], and [=readonly-storage-buffer=] bindings, 
+    indicates that the binding has a dynamic offset. One offset must be passed to 
+    setBindGroup for each dynamic binding in increasing order of
+    {{GPUBindGroupLayoutBinding/binding}} number.
+
+### {{GPUDevice/createBindGroupLayout()|GPUDevice.createBindGroupLayout(GPUBindGroupLayoutDescriptor)}} ### {#GPUDevice-createBindGroupLayout}
+
+A valid {{GPUBindGroupLayout}} object has an <dfn>internal map</dfn> of |u32| <dfn>binding number</dfn>s to internal <dfn>layout binding</dfn> objects. 
+Each [=layout binding=] has a {{GPUShaderStageFlags}} <dfn>visibility</dfn> bitset and a {{GPUBindingType}} <dfn>type</dfn>.
```

Comment:
I didn't link to {{GPUBindGroupLayoutBinding/visibility}} here because I'm not necessarily referring to that field of the `GPUBindGroupLayoutDescriptor` here. Instead I want to make it clear that the `GPUBindGroupLayout` implementation has its own internal way of tracking `visibility` for each `GPUBindGroupLayoutBinding` it was created with, that doesn't necessarily have to be of the same structure as `GPUBindGroupLayoutBinding`.

Reply:
Addressed this by better describing `GPUBindGroupLayout`'s internal "slots".